client/checks.go

```diff
@@ -172,3 +172,38 @@ func (client *Client) doMultiCheckAction(actionName string, checks []string) (*C
 
 	return results, nil
 }
+
+type RefreshCheckOptions struct {
+	// Name of the check to refresh, required.
+	Name string
+}
+
+type RefreshCheckResult struct {
+	Info *CheckInfo `json:"info"`
```

Comment:
Should this be a non-pointer, seeing it's always present, and essentially embedding the CheckInfo in this struct?
```suggestion
	Info CheckInfo `json:"info"`
```